### PR TITLE
Fixes classnames in inline documentation

### PR DESCRIPTION
**Pull request by gguuss**

### CODE REVIEW

File: iot/api-client/manager/src/main/java/com/example/cloud/iot/examples/DeviceRegistryExample.java [hunk 1]
```diff
@@ -84,11 +84,19 @@
  * <code>
  * $ mvn clean compile assembly:single
  * $ mvn exec:java \
```
Comment (lesv) [inline iot/api-client/manager/src/main/java/com/example/cloud/iot/examples/DeviceRegistryExample.java]: We've found that some libraries don't like exec:java as the shutdown code isn't called properly.  We are moving to exec:exec.

File: iot/api-client/manager/src/main/java/com/example/cloud/iot/examples/MqttExample.java [hunk 2]
```diff
@@ -57,12 +57,14 @@
  *
  * <pre>
  *   $ mvn compile
- *   $ mvn exec:java -Dexec.mainClass="com.google.cloud.iot.examples.MqttExample" \
- *       -Dexec.args="-project_id=my-project-id \
- *       -registry_id=my-registry-id \
- *       -device_id=my-device-id \
- *       -private_key_file=/path/to/private_pkcs8 \
- *       -algorithm=RS256"
+ *   $ mvn exec:java \
```
Comment (lesv) [inline iot/api-client/manager/src/main/java/com/example/cloud/iot/examples/MqttExample.java]: dito

File: iot/api-client/manager/src/main/java/com/example/cloud/iot/examples/DeviceRegistryExample.java (hunk 1 above)
Comment (gguuss) [reply iot/api-client/manager/src/main/java/com/example/cloud/iot/examples/DeviceRegistryExample.java]: SGTM, will resolve once updated.

**Comment by gguuss** [comment]
I'll merge once verifying all the commands work as expected.

**Comment by gguuss** [comment]
Looks like a few updates will be necessary :)

**Comment by gguuss** [comment]
@lesv I'm concerned that moving to `mvn exec:exec` is adding a lot of complexity to the sample. To translate 1 of the 3 command line runners (the simple one) to `exec:exec` has added 100 lines of XML and required careful revision to the run arguments because it does not support passing a complete `ARGV` to the main Java class and instead requires each individual parameter be passed as variables to `mvn exec`.

For this PR, I'll be skipping translating the other samples for IoT to this pattern as it will be too much code for a sensible PR.

**Comment by lesv** [comment]
@gguuss I wasn't expecting you to do all of them.  I understood it was annoying and difficult.  #1524 was opened for this process.

**Comment by gguuss** [comment]
> #1524 was opened for this process.

SGTM, I'll start keeping some notes :) I understand a few advantages of `exec:exec` it's just a little more verbose.

**Pull request merged**